packages/client/src/devices/MicrophoneManager.ts

```diff
@@ -24,6 +26,9 @@ export class MicrophoneManager extends InputMediaDeviceManager<MicrophoneManager
       if (callingState !== CallingState.JOINED) {
         if (callingState === CallingState.LEFT) {
           await this.stopSpeakingWhileMutedDetection();
+          if (isReactNative()) {
+            this.rnSpeechDetector?.stop();
+          }
```

Comment:
I think this block is redundant. The detector will be stopped in `soundDetectorCleanup()`.

Reply:
Done